DFC.Integration.AVFeed/DFC.Integration.AVFeed.Function.Common/DFC.Integration.AVFeed.Function.Common.csproj

```diff
@@ -99,7 +99,7 @@
     <Compile Include="Properties\AssemblyInfo.cs" />
   </ItemGroup>
   <ItemGroup>
-    <None Include="app.config" />
+    <None Include="app.template" />
```

Comment:
this should be renamed as app.config.template to keep it inline with dfc-digital